overrides/netlify.toml

```diff
@@ -21,6 +21,7 @@
     Content-Security-Policy = """
       default-src 
         'self'
+        https://*.dcube.cloud
```

Comment:
needs `/` at back?

Reply:
Fixed in https://github.com/opengovsg/isomer-build/pull/48/commits/09c642abd46fef479de56b26cfef474bfbd67d32!